### PR TITLE
CVSB-10163: components with unit tests with rebase with common

### PR DESCRIPTION
**Pull request by m-ujaffer**
CVSB-10163: PR changes for merge to common. rebased after merge to common.

### CODE REVIEW

File: src/app/technical-record/tech-rec-history/tech-rec-history.component.spec.ts [hunk 1]
```diff
@@ -1,76 +1,46 @@
...
-      'vin': 'XMGDE02FS0H012345',
-      'vehicleSize': 'small',
-      'testStationName': 'Rowe, Wunsch and Wisoky',
-      'vehicleId': 'JY58FPP',
-      'vehicleType': 'psv',
-      'axles': [
-        { 'parkingBrakeMrk': false, 'axleNumber': 1 },
-        { 'parkingBrakeMrk': true, 'axleNumber': 2 },
-        { 'parkingBrakeMrk': false, 'axleNumber': 3 }
+    component.techRecordsJson = {
```
Comment (JonLazariniDD) [inline src/app/technical-record/tech-rec-history/tech-rec-history.component.spec.ts:22]: can we use a mock for that?

File: src/app/technical-record/vehicle-summary/vehicle-summary.component.html [hunk 2]
```diff
@@ -8,30 +8,37 @@
...
-  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType == 'trl'">
+  <div class="govuk-summary-list__row">
     <dt class="govuk-summary-list__key">
-      First use date
+      Year of manufacture
     </dt>
-    <dd id="test-firstUseDate" class="govuk-summary-list__value">
-      {{ activeRecord.firstUseDate | DefaultNullOrEmpty }}
+    <dd id="test-manufactureYear" class="govuk-summary-list__value">
+      {{ activeRecord.manufactureYear | DefaultNullOrEmpty }}
```
Comment (JonLazariniDD) [inline src/app/technical-record/vehicle-summary/vehicle-summary.component.html:41]: do we need data pipe here?

Comment (m-ujaffer) [reply src/app/technical-record/vehicle-summary/vehicle-summary.component.html:41]: this is just a numeric value.

File: src/app/technical-record/tech-rec-history/tech-rec-history.component.spec.ts (hunk 1 above)
Comment (m-ujaffer) [reply src/app/technical-record/tech-rec-history/tech-rec-history.component.spec.ts:23]: fixed

**Pull request merged**